### ISSUE TITLE
Update font blacklist

### ISSUE THREAD

**Issue by classilla** [classilla/tenfourfox#566]
From https://tenfourfox.tenderapp.com/discussions/problems/8844-waste-management-have-broken-themselves-on-tenfourfox-with-their-new-and-improved-yeah-right-website

**Comment by classilla** [comment]
Tested working

**Comment by chris-chtrusch** [comment]
https://www.kulturstiftung-des-bundes.de/typo3conf/ext/base_ksb/Resources/Public/38c1bdeb69b2cae2f59fae38f127aa6d.woff2

https://www.kulturstiftung-des-bundes.de/typo3conf/ext/base_ksb/Resources/Public/9437aea92626957f9d7fd0e5f548b689.woff2

**Comment by classilla** [comment]
Okay, will add for FPR17.

**Comment by chris-chtrusch** [comment]
Is it possible to blacklist https://www.kulturstiftung-des-bundes.de/typo3conf/ext/base_ksb/Resources/Public/*.woff ?
They seem to switch this part of the URL every once in a while.

**Comment by classilla** [comment]
Yes, I'll change the entry. It's designed so that it can match on partial URLs.

**Comment by chris-chtrusch** [comment]
https://cdn.trustpilot.net/brand-assets/2.1.0/fonts/trustpilot-default-font-regular.woff2
https://cdn.trustpilot.net/brand-assets/2.1.0/fonts/trustpilot-default-font-medium.woff2
https://cdn.trustpilot.net/brand-assets/2.1.0/fonts/trustpilot-default-font-bold.woff2

**Comment by classilla** [comment]
I'll pick this up for FPR21 since FPR20 is already halfway through build and certify.

**Comment by chris-chtrusch** [comment]
Trustpilot still has these font issues.

**Comment by classilla** [comment]
Damn it, this slipped. I'll get for final.

**Comment by classilla** [comment]
New sets landed, tested working.

**Comment by chris-chtrusch** [comment]
https://hartzfacts.de/google-fonts/s/notoseriftc/v7/XLY9IZb5bJNDGYxLBibeHZ0Bvr8vXX5MTw.woff2

https://hartzfacts.de/google-fonts/s/notoseriftc/v7/XLY9IZb5bJNDGYxLBibeHZ0BvtssXX5MTw.woff2

https://hartzfacts.de/google-fonts/s/notoseriftc/v7/XLYgIZb5bJNDGYxLBibeHZ0BtnAOSA.woff2

https://hartzfacts.de/google-fonts/s/notoseriftc/v7/XLY9IZb5bJNDGYxLBibeHZ0BvoMtXX5MTw.woff2

https://hartzfacts.de/google-fonts/s/notoseriftc/v7/XLY9IZb5bJNDGYxLBibeHZ0BvvMpXX5MTw.woff2

**Comment by classilla** [comment]
Ack'ed for FPR25

**Comment by chris-chtrusch** [comment]
I found two more fonts to add to the font list. For my own build I did this at gfxPlatformMac.cpp:

		EXACT_URL("https://som.yale.edu/themes/custom/som/fonts/neuehaasunica/NeueHaasUnicaBlackItalic/italic_normal_subset1.woff2");

		EXACT_URL("https://som.yale.edu/themes/custom/som/fonts/neuehaasunica/NeueHaasUnicaBlack/normal_normal_subset1.woff2"); 

This worked, but EXACT_URL is probably the least generic/desirable place to add it.

**Comment by classilla** [comment]
How about

```
HTTP_OR_HTTPS_SUBDIR("som.yale.edu","/themes/custom/som/fonts/neuehaasunica/NeueHaasUnicaBlack");
```

Or does that block too many things?

I'm getting together another patch drop, so I'll include this blocklist update.

**Comment by chris-chtrusch** [comment]
That should work as well. This being my first try to manipulate the font list I simply didn't know how to use the more generic methods. 

Should I keep posting fonts here or is there a way to contribute directly, i.e. submit changes for review? The Unofficial TenFourFox Development Toolkit I use to build the browser doesn't have full Git functionality, I can merge my local changes when I re-download the source, but cannot "push" my changes upwards.

**Comment by classilla** [comment]
Officially the only way Github does it is to set up a full pull request, which isn't ideal. Oh well.

Short of manually posting URLs, one other option is to do a `git diff` with your changes and post the diff. I'm okay with either.

**Comment by classilla** [comment]
You may want to remove your local change before pulling or you'll get a merge conflict, but this is now upstreamed.

**Comment by chris-chtrusch** [comment]
https://www.washingtonpost.com/wp-stat/assets/fonts/PostoniWide-Regular.woff2
https://www.washingtonpost.com/wp-stat/assets/fonts/PostoniWide-Bold.woff2

**Comment by chris-chtrusch** [comment]
Major liberal German Newpaper did a site re-design without consulting TenFourFox.

https://assets.tagesspiegel.de/fonts/Abril_Text_Regular-basic.c371b6fb87298eb4990a97124a47e7e4.woff2

https://assets.tagesspiegel.de/fonts/Abril_Text_Regular-extended.413ee6f014cfa7c05f2979ccb6498cee.woff2

https://assets.tagesspiegel.de/fonts/Abril_Text_Italic-basic.6c5932e5019579d6c76d1822e5aacc4d.woff2

https://assets.tagesspiegel.de/fonts/Abril_Text_Bold-extended.7e1a83a609780222181670109982a7a2.woff2

https://assets.tagesspiegel.de/fonts/Abril_Text_Bold-basic.ed72f8ea0597f63d7cc1a47ae7bb3acd.woff2

https://assets.tagesspiegel.de/fonts/Abril_Text_BoldItalic-basic.1cd71e407d313b5db30f65b5a9bc639f.woff2

https://assets.tagesspiegel.de/fonts/franklingothic-book-extended.b0f1fe099d20b0de899ed94f6292ba63.woff2

https://assets.tagesspiegel.de/fonts/franklingothic-book-other.6c8ff800d4fec0ff804a756fc400d2b5.woff2

https://assets.tagesspiegel.de/fonts/franklingothic-book-basic.c62561d04e5eb023ca3e2fde512605c0.woff2

https://assets.tagesspiegel.de/fonts/franklingothic-med-extended.e84bf4c79c0055f692e535db121cff4a.woff2

https://assets.tagesspiegel.de/fonts/franklingothic-med-basic.4262342c15da29dbdffaff795d31ad13.woff2

**Comment by classilla** [comment]
Noted, thank you. Probably will do a regular update around now too.

**Comment by chris-chtrusch** [comment]
https://www.tagesschau.de/resources/assets/fonts/TheSansC5s-4_SemiLightItalic.woff2
https://www.tagesschau.de/resources/assets/fonts/TheSansC5s-4_SemiLight.woff2
https://www.tagesschau.de/resources/assets/fonts/TheSansC5s-6_SemiBold.woff2

**Comment by classilla** [comment]
Noted and landing shortly
